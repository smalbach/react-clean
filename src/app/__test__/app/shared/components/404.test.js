import React from "react";
import { mount } from "enzyme";
import Error404 from "../../../../shared/components/404";

import { create } from "react-test-renderer";

describe("<Error404>", () => {
  const error = mount(<Error404 />);
  test("Render of footer", () => {
    expect(error.length).toEqual(1);
  });

  test("Error snapshot", () => {
    const error = create(<Error404 />);
    expect(error).toMatchSnapshot();
  });
});
