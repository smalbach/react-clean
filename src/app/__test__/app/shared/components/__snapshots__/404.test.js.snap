// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error404> Error snapshot   1`] = `
<div
  className="fade alert alert-danger show"
  role="alert"
>
  <div
    className="alert-heading h4"
  >
    Opps! You got an error!
  </div>
  <p />
</div>
`;

exports[`<Error404> Error snapshot 1`] = `
<div
  className="fade alert alert-danger show"
  role="alert"
>
  <div
    className="alert-heading h4"
  >
    Opps! You got an error!
  </div>
  <p />
</div>
`;
